generators/utils.js

```diff
@@ -513,10 +513,11 @@ function getEnumInfo(field, clientRootFolder) {
   const customValuesState = getCustomValuesState(enums);
   return {
     enumName: fieldType,
+    javadoc: field.fieldTypeJavadoc && this.getJavadoc(field.fieldTypeJavadoc, 0),
```

Comment:
why the 0?

Reply:
That is the indentSize.
Not sure if this has any effect as overall generated file seems formatted (with prettier?)

Reply:
you can always test :) try using 2 instead of 0, or see if 0 can be the default

but if I'm not mistaken, the indent level is important

Reply:
I just tested :), indent level matters only if we are generating without prettier (i.e. with  `--skip-prettier`). by default it prettifies all the files with proper indentation.
To cover that case I changed the indents to 4 for field javadoc and default to zero for class level javadoc.
Thanks for pointing out.